src/Apps/Sell/Routes/ThankYouRoute.tsx

```diff
@@ -43,22 +43,26 @@ export const ThankYouRoute: React.FC<ThankYouRouteProps> = props => {
           <Join separator={<Spacer y={4} />}>
             <Join separator={<Spacer y={2} />}>
               <SubmissionStepTitle>
-                {isSubmitted
-                  ? "Thank you for submitting your artwork"
+                {!!isSubmitted
```

Comment:
No need to convert `isSubmitted` to a boolean here. It's just recommended when using `&&` because `isSubmitted` could be a falsy value that would be rendered (like `0`).

```suggestion
                {isSubmitted
```